util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
+    try:
+        if value is None:
+            return default
+        if value.lower() == 'true':
+            return True
+        elif value.lower() == 'false':
+            return False
+        else:
+            return default
+    except Exception as e:
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rkj6124/test/pull/2#issuecomment-2425916327>#ac3cc5</a><span> - 10/21/2024, 08:09 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The code is catching a broad 'Exception'. It's generally better to catch specific exceptions that you expect might occur.
</div>
<div id="fix">
<b>Fix:</b> Catch more specific exceptions that might occur during the string conversion process.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -9,5 +9,5 @@
          if value.lower() == 'false':
              return False
          return default
 -    except Exception as e:
 +    except (AttributeError, ValueError):
          return default
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged